src/components/HomePage/QuickInfoArea/InfoBox.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import homePageStyles from '../HomePage.module.css';
+
+import Typography from '@material-ui/core/Typography';
+
+class InfoBox extends Component {
```

Comment:
Convert to functional component to reduce boilerplate and use object destructuring to get the props

Reply:
Currently in the process of migrating existing components into functional ones

Reply:
:shipit: